tests/resources/Harbor-Pages/OIDC_Auth.robot

```diff
@@ -0,0 +1,37 @@
+# Copyright Project Harbor Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  This resource provides any keywords related to the Harbor private registry appliance
+Resource  ../../resources/Util.robot
+
+*** Variables ***
+${HARBOR_VERSION}  v1.1.1
+
+*** Keywords ***
+
+Sign In Harbor With Ldap User
+    [Arguments]  ${url}  ${username}=${OIDC_USERNAME}
```

Comment:
add some comments for the sign in workflow? Someone may not have a background about it.